docker/recorder/src/test/java/org/arquillian/cube/docker/impl/client/reporter/TakeDockerEnvironmentTest.java

```diff
@@ -131,4 +165,48 @@ public void should_report_schema_of_docker_composition() {
         assertThat(screenshotEntry.getLink()).isEqualTo("reports/schemas/docker_composition.png");
     }
 
+   @Test
+    public void should_report_log_file(){
+        final TakeDockerEnvironment takeDockerEnvironment = new TakeDockerEnvironment();
+        takeDockerEnvironment.propertyReportEvent = propertyReportEvent;
+        takeDockerEnvironment.reportContainerLogs(new BeforeStop(CUBE_ID), dockerClientExecutor, new ReporterConfiguration());
+        verify(propertyReportEvent).fire(propertyReportEventArgumentCaptor.capture());
+
+        final PropertyReportEvent propertyReportEvent = propertyReportEventArgumentCaptor.getValue();
+        final PropertyEntry propertyEntry = propertyReportEvent.getPropertyEntry();
+        assertThat(propertyEntry).isInstanceOf(FileEntry.class);
+
+        FileEntry fileEntry = (FileEntry) propertyEntry;
+        assertThat(fileEntry.getPath()).isEqualTo("target/reports/logs/tomcat.log");
+    }
+
+    @Test
```

Comment:
@bartoszmajsak , @lordofthejars To construct network Object as I am interested in only rx_bytes & tx_bytes, If I use only those values instead of using all values by keeping schema same. 
e.g. Instead of constructing entire object
`{eth0={rx_bytes=724, rx_packets=7, rx_errors=0, rx_dropped=0, tx_bytes=418, tx_packets=5, tx_errors=0, tx_dropped=0}}`

I can construct only
`{eth0={rx_bytes=724,tx_bytes=418}}`

 WDYT, will this be a good idea?